packages/ani-cli/build.sh

```diff
@@ -5,10 +5,12 @@ TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=2.2
 TERMUX_PKG_SRCURL=https://github.com/pystardust/ani-cli/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=973d335a75bd7f920c244000ad6b057f702fb37752e7bea1b5bcf038785ec925
-TERMUX_PKG_DEPENDS="aria2, curl, grep, mpv, openssl-tool, sed, gawk, ffmpeg"
+TERMUX_PKG_DEPENDS="aria2, curl, grep, openssl-tool, sed, gawk, ffmpeg"
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
 termux_step_make_install() {
 	install -Dm700 -t $TERMUX_PREFIX/bin ani-cli
+	echo 'am start --user 0 -a android.intent.action.VIEW -d "$1" -n is.xyz.mpv/.MPVActivity' > $PREFIX/bin/mpv
```

Comment:
`$TERMUX_PREFIX` should be used instead of `$PREFIX` in these build scripts

Reply:
For me locally, `$TERMUX_PREFIX` is unset by default, where `$PREFIX` points to the relevant directory.
Does `$TERMUX_PREFIX` get set accordingly during a pkg run?
Maybe you can point me to relevant docs

Reply:
In the environment on device, PREFIX will be set. When building packages with the [build-package.sh script](https://github.com/termux/termux-packages/blob/master/build-package.sh), we try to stick to using TERMUX_PREFIX, which is set in [scripts/properties.sh](https://github.com/termux/termux-packages/blob/master/scripts/properties.sh#L29), which is [sourced in build-package.sh](https://github.com/termux/termux-packages/blob/master/build-package.sh#L313)

Reply:
Also remember to chmod the script or else it cannot be run from ani-cli